pysteps/nowcasts/steps.py

```diff
@@ -33,6 +33,863 @@
     DASK_IMPORTED = False
 
 
+class StepsNowcaster:
+    def __init__(self, precip, velocity, timesteps, **kwargs):
+        # Store inputs and optional parameters
+        self.precip = precip
+        self.velocity = velocity
+        self.timesteps = timesteps
+        self.n_ens_members = kwargs.get("n_ens_members", 24)
+        self.n_cascade_levels = kwargs.get("n_cascade_levels", 6)
+        self.precip_thr = kwargs.get("precip_thr", None)
+        self.kmperpixel = kwargs.get("kmperpixel", None)
+        self.timestep = kwargs.get("timestep", None)
+        self.extrap_method = kwargs.get("extrap_method", "semilagrangian")
+        self.decomp_method = kwargs.get("decomp_method", "fft")
```

Comment:
```suggestion
        self.decomposition_method = kwargs.get("decomposition_method", "fft")
```
If this works for you, too, then we should change it in the rest of the code as well.